packages/vsx-registry/src/browser/vsx-extension.tsx

```diff
@@ -322,18 +322,23 @@ const downloadCompactFormatter = new Intl.NumberFormat(undefined, { notation: 'c
 export class VSXExtensionComponent extends AbstractVSXExtensionComponent {
     render(): React.ReactNode {
         const { iconUrl, publisher, displayName, description, version, downloadCount, averageRating } = this.props.extension;
-        return <div className='theia-vsx-extension'>
-            {iconUrl ?
+        const extensionsContainer = document.getElementById('vsx-extensions-view-container');
```

Comment:
it is an anti pattern to rely on DOM to render Virtual DOM

the proper flow:
- render Virtual DOM
- let react to work with DOM